examples/oauth-proxy/readme.md

```diff
@@ -0,0 +1,70 @@
+# OAuth Proxy APIm policies
```

Comment:
Can you change all "APIm" references to Azure API Management please?

Reply:
Done